### PR TITLE
address golint warnings for package control/plugin

### PR DESCRIPTION
**Pull request by candysmurf**

### CODE REVIEW

File: control/plugin/metric.go [hunk 1]
```diff
@@ -81,13 +87,14 @@ func (p *PluginConfigType) GobDecode(data []byte) error {
 	return nil
 }
 
+// NewPluginConfigType retruns a new instance of PluginConfigType
```
Comment (tiffanyfj) [inline control/plugin/metric.go]: returns

File: control/plugin/metric.go [hunk 2]
```diff
@@ -129,17 +136,17 @@ func NewPluginMetricType(namespace []string, timestamp time.Time, source string,
...
 }
 
-// Returns the last time this metric type was received from the plugin.
+// LastAdvertisedTime returns the last time this metric type was received from the plugin.
 func (p PluginMetricType) LastAdvertisedTime() time.Time {
 	return p.LastAdvertisedTime_
 }
 
-// Returns the namespace.
+// Version returns the namespace.
```
Comment (tiffanyfj) [inline control/plugin/metric.go]: Version returns the metric version.

File: control/plugin/plugin.go [hunk 3]
```diff
@@ -83,6 +92,8 @@ var (
 	}
 )
 
+// Plugin interface defines the types and methods
+// that a plugin must implements
```
Comment (tiffanyfj) [inline control/plugin/plugin.go]: implement

File: control/plugin/processor_test.go [hunk 4]
```diff
@@ -81,7 +81,7 @@ func (s *MockProcessorSessionState) Token() string {
 	return s.token
 }
 
-func (m *MockProcessorSessionState) ResetHeartbeat() {
+func (s *MockProcessorSessionState) ResetHeartbeat() {
```
Comment (tiffanyfj) [inline control/plugin/processor_test.go]: Since your other mock examples started with m this should be ms so it's clear it's a mock.

**Comment by candysmurf** [comment]
Closing for higher priority items. We'll get back to it later.

**Pull request closed without merging**